src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1089,12 +1167,12 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
             building.onWorldTick(event);
         }
 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "world" is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)